storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/common/JdbcTemplater.java

```diff
@@ -22,6 +22,12 @@
 
 public interface JdbcTemplater extends Serializable {
   static String getTableName(FeatureSetProto.FeatureSetSpec featureSetSpec) {
+    System.out.println(
```

Comment:
is this the convention?  Isn't there a proper logger?

Reply:
We added:        
```
 <dependency>
		    <groupId>org.slf4j</groupId>
		    <artifactId>slf4j-jdk14</artifactId>
		    <version>1.7.25</version>
		</dependency>
``` 
for a better logger under jdbc pom.xml